geoprocessing/src/migrations/geoprocessing/1620408879000-AddUniqueConstraintsToAdminRegionsTable.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUniqueConstraintsToAdminRegionsTable1620408879000
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+CREATE UNIQUE INDEX unique_l0_regions ON admin_regions (gid_0, level) where level = 'country';
+CREATE UNIQUE INDEX unique_l1_regions ON admin_regions (gid_1, level) where level = 'adm_1';
+CREATE UNIQUE INDEX unique_l2_regions ON admin_regions (gid_2, level) where level = 'adm_2';
```

Comment:
As far as I remember, indexed value has a length limit (of `varchar(2712)`) thus we could enforce max length as well to avoid unpredicted fails on inserts.